src/Tizen.Network.Stc/Interop/Interop.Libraries.cs

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
```

Comment:
Please update the year in each file to 2019.

Reply:
Done